src/m3u8/m3u8-parser.js

```diff
@@ -375,7 +375,8 @@
 
     // the manifest is empty until the parse stream begins delivering data
     this.manifest = {
-      allowCache: true
+      allowCache: true,
+      discontinuityStarts: []
```

Comment:
so `discontinuityStarts` end up being a list of indexes pointing at segments that begin a discontinuity?